<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComposeWidget</class>
 <widget class="QWidget" name="ComposeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compose Mail</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fromLabel">
     <property name="text">
      <string>From</string>
     </property>
     <property name="buddy">
      <cstring>sender</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="sender"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="mailText"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="sendButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="subjectLabel">
     <property name="text">
      <string>Subject</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="subject"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
